.github/workflows/dependabot-automerge.yml

```diff
@@ -0,0 +1,37 @@
+name: dependabot merger
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  review-pr:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' &&
+      github.event_name == 'pull_request' }}
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - name: dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.3.1
```

Comment:
I just noticed that this action seems to be rather old:

![image](https://github.com/SAP/btp-environment-variable-access/assets/9248711/6396df05-522d-4f8e-af81-295d30b51c0b)


I guess this might actually be a problem as GitHub won't support those deprecated methods and stacks much longer.